Source/SwiftLintFramework/Models/Example.swift

```diff
@@ -73,6 +88,32 @@ public extension Example {
     func removingViolationMarkers() -> Example {
         return with(code: code.replacingOccurrences(of: "↓", with: ""))
     }
+}
+
+extension Example {
+    func skipWrappingInCommentTest() -> Example {
+        var new = self
+        new.testWrappingInComment = false
+        return new
+    }
+
+    func skipWrappingInStringTest() -> Example {
+        var new = self
+        new.testWrappingInString = false
+        return new
+    }
+
+    func skipMultiByteOffsetTest() -> Example {
+        var new = self
+        new.testMultiByteOffsets = false
+        return new
+    }
+
+    func skipDisableCommandTest() -> Example {
+        var new = self
+        new.testDisableCommand = false
+        return new
+    }
 
     /// Makes the current example focused.
     func focused() -> Example {
```

Comment:
```
Source/SwiftLintFramework/Models/Example.swift:119:10: error: Unused Declaration Violation: Declarations should be referenced at least once within all files linted. (unused_declaration)
```

Reply:
I think we can just silence this one since it's designed to be a debug helper for local development.

Reply:
Silenced the warning and added a comment to make the intention clear.